assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -125,6 +129,372 @@ export default {
         });
       });
     });
+
+    withPluginApi("1.3.0", (api) => {
```

Comment:
I think it'll also be nicer from an organisation perspective if we move the sidebar specific stuff into another initializer.